app/services/search/buffer_suggestions_builder.rb

```diff
@@ -9,7 +9,16 @@ def initialize(location, search_params)
 
   def get_buffers_suggestions
     buffer_vacancy_count = LocationPolygon.with_name(location).buffers.map do |buffer|
-      [buffer.first, Search::AlgoliaSearchRequest.new(search_params.merge(polygon: [buffer.last])).stats.last]
+      # In this step (2) of the LocationPolygon refactor, the format of buffers will be different
+      # before and after running the import task. Before it's a 1D array; after, it's 2D. So I am temporarily
+      # rewriting this method to cope with both formats. This will be reverted in step 3.
+      polygon = if buffer.last.first.is_a?(Float)
```

Comment:
`buffer.last.first.is_a?(Float)` for a 2D array? What is `buffer.last.first` for a 1D array? array -> float -> ?

Reply:
```
buffer => { "5" => [] }
buffer.last => []
buffer.last.first => Float or [Float Float Float]
```